e2e/files.spec.ts

```diff
@@ -140,10 +150,7 @@ const uploadFile = async (
   await testAuth.step('ensure there was not an upload error', async () => {
     const uploadError = page.getByRole('heading', { name: 'Upload Error' })
     if (await uploadError.isVisible()) {
-      const uploadErrorText = await page
-        .getByRole('dialog')
-        .getByText('Unable to upload the file')
-        .textContent()
+      const uploadErrorText = await page.getByRole('dialog').textContent()
```

Comment:
Output the upload error text for any type of upload error, not only errors associated with the MultipartUploader